src/components/Auth/SignIn.tsx

```diff
@@ -44,6 +45,23 @@ const SignIn: React.FC = () => {
     console.warn(error)
   }, [error])
 
+  const requestConfirmationCode = () => {
+    if (formValues.username.length === 0) {
+      // eslint-disable-next-line no-console
+      console.log("user name missing")
+    } else {
+      forgotPassword(formValues.username)
+        .then(
+          // eslint-disable-next-line no-console
+          data => console.log(data)
+        )
+        .catch(err => {
+          // eslint-disable-next-line no-console
+          console.log(err)
+        })
```

Comment:
Why do we need these `console.log` statements?

Reply:
mistake